src/components/Expenses/ExpenseDate.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./ExpenseDate.css";
+
+interface ExpenseDateProps extends React.HTMLAttributes<JSX.Element> {
+  date?: Date
+}
+
+function ExpenseDate(props: ExpenseDateProps): JSX.Element {
+  const month: string =
```

Comment:
IMO it would be nice to extract the logic of parsing date to the human readable format to a separate (pure) function.
Something like:
```
function parseDate(data: Date) {
...
```

then it would be more reusable and unit testable